docs/application/native/guides/complications/watchface-complication-provider.md

```diff
@@ -195,6 +213,15 @@ And, the specific default value is mandatory depending on each support types:
 | `<time-type>` | `<default-hour>` | `<default-short-text>` |
 |               | `<default-minute>` | `<default-extra-data>` |
 |               | `<default-second>` |         |
+|               | `<default-timezone-id>` |         |
+
+
+> **Note**
+>
+> `<default-hour>`, `<default-minute>`, and `<default-second>` are deprecated since Tizen 5.5.
+> Instead, use `<default-timezone-id>`.
+> `<default-timezone-id>` is the value that declared in TZ database. For example, Asia/Seoul.

```

Comment:
Suggested rephrase for the note:

>**Note**
>
>Use `<default-timezone-id>`, as `<default-hour>`, `<default-minute>`, and `<default-second>` are deprecated since Tizen 5.5. 
> `<default-timezone-id>` is the value that is declared in TZ database. For example, Asia/Seoul.
> If the XML contains `<default-timezone-id>`, then `<default-hour>`, `<default-minute>`, and `<default-second>` are not necessary.

Reply:
@rishu-samsung I have a suggestion to you about this comment.
As this note gives information which values are deprecated, and which values are to be used it instead, how about keeping original order in the sentence?

I think for this case, letting a user know which values should be replaced first is better. If the user was not using the value, the next sentence can be ignored. If the user was using that deprecated, values, the next sentence will be read carefully to get the information of new value which is replacing it.

Reply:
@gotoextreme 
No need to change as @mijin85cho suggested. 
just remove **that** after value 
Original: 
>**Note**
>
> `<default-hour>`, `<default-minute>`, and `<default-second>` are deprecated since Tizen 5.5.
>> Instead, use `<default-timezone-id>`.
> `<default-timezone-id>` is the value declared in TZ database. For example, Asia/Seoul.